app/controllers/state_file/questions/return_status_controller.rb

```diff
@@ -23,26 +19,6 @@ def prev_path
 
       private
 
-      def title
```

Comment:
moved some of these into the view since we should try to keep text there if we can